language/diem-vm/src/diem_transaction_executor.rs

```diff
@@ -198,8 +185,6 @@ impl DiemVM {
                 )
                 .map_err(|e| e.into_vm_status())?;
 
-            charge_global_write_gas_usage(cost_strategy, &session)?;
```

Comment:
I think this should stay no? We still want to keep charging for global writes is my understanding?